lib/wcc/contentful/model_builder.rb

```diff
@@ -59,13 +59,14 @@ def build_model(typedef)
             bad_fields = filter.keys.reject { |k| fields.include?(k) }
             raise ArgumentError, "These fields do not exist: #{bad_fields}" unless bad_fields.empty?
 
-            if defined?(context[:preview]) && context[:preview] == ENV['CONTENTFUL_PREVIEW_PASSWORD']
-              result = WCC::Contentful::Model.preview_store.find_by(content_type: content_type, filter: filter)
-            else
-              result = WCC::Contentful::Model.store.find_by(content_type: content_type, filter: filter)
-            end
+            result =
+              if defined?(context[:preview]) && context[:preview] == ENV['CONTENTFUL_PREVIEW_PASSWORD']
```

Comment:
I think I'd prefer it if the gem didn't care at all about the `CONTENTFUL_PREVIEW_PASSWORD`.  Let each app decide what's the best way to enable previews or not - if the app passes `preview: true` into the context, then it should get the preview.